sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/implementation/EntityHelper.java

```diff
@@ -622,4 +638,177 @@ public interface TopicAccessor {
          */
         void setName(TopicProperties topicProperties, String name);
     }
+
+    /**
+     * Converts a Response into its corresponding {@link QueueDescriptionFeedImpl} then mapped into {@link
+     * QueueProperties}.
+     *
+     * @param response HTTP Response to deserialize.
+     * @param logger The ClientLogger logging errors and warnings.
+     * @return The corresponding HTTP response with convenience properties set.
+     */
+    public static Response<QueueDescriptionFeedImpl> deserializeQueueFeed(Response<Object> response,
```

Comment:
Why creating multiple named methods instead of `<T>` like it did before?

Reply:
These already had separate methods like these in `AdministrationModelConverter`, I just moved these into `EntityHelper` as they are used in multiple locations.